tests/python/gaia-ui-tests/gaiatest/tests/unit/test_override_defaults.py

```diff
@@ -0,0 +1,20 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from gaiatest import GaiaTestCase
+from gaiatest import DEFAULT_SETTINGS
+
+TEST_DEFAULTS = {'ftu.manifestURL': 'app://ftu.gaiamobile.org/manifest.webapp'}
+
+
+class TestOverrideDefaults(GaiaTestCase):
+
+    def set_defaults(self):
```

Comment:
OK, this works.

I probably would have taken a slightly different approach of having:

  def modifySettings(self, settings):
       return settings

...on GaiaTestCase.

Then from GaiaTestCase.setDefaults(self)--note I took out the defaults=none--line 941, after the testvars update

  defaults = self.modifySettings(defaults)

That would let the child testcase override as follows:

  class TestOverrideDefaults(GaiaTestCase):

      def modifySettings(self, settings):
          settings.update(TEST_DEFAULTS)
          return settings

That takes out having to have logic about whether or not you sent in different defaults, and takes away any chance that the parent will stomp your test's defaults by adding something to its own implementation, so long as it guarantees you're called last in the modification chain. As it stands, you're called first by necessity.

This (a functional abstract method, basically) is a pretty standard way to add an override when you have doA, doB, dC in a method, and only want to override doB.